DOCS/man/options.rst

```diff
@@ -1731,6 +1730,13 @@ Video
     This is usually only needed with broken GPUs, where a codec is reported
     as supported, but decoding causes more problems than it solves.
 
+    .. note::
+
+        Depending on the configurations of GPU hardware and driver, probing
+        for codecs which the GPU does not support can unnecessarily slow down
+        video playback initialization. To alleviate this, explicitly specify
+        a list which only includes the codecs supported on the setup.
```

Comment:
in wm4 fashion this could be reworded to blame it on nvidia ("On some broken drivers (NVIDIA), probing [...]"). I have at least never heard of this happening.